book/chrome.md

```diff
@@ -416,8 +416,7 @@ objs = [obj for obj in tree_to_list(self.document, [])
 ```
 
 In principle there might be more than one layout object in this
-list.^[This is actually not possible in our browser at this point, but
-in real browsers there are all sorts of ways this could happen, like
+list.^[In real browsers there are all sorts of ways this could happen, like
```

Comment:
It's not true that it's not possible: typically if a node is in the list, so are its parents. It's also not a relevant point here.